sample/src/androidTest/java/com/schibsted/spain/barista/sample/PartiallyVisibleViewPagerTest.java

```diff
@@ -0,0 +1,57 @@
+package com.schibsted.spain.barista.sample;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.schibsted.spain.barista.assertion.BaristaVisibilityAssertions.assertDisplayed;
+import static com.schibsted.spain.barista.assertion.BaristaVisibilityAssertions.assertNotDisplayed;
+import static com.schibsted.spain.barista.interaction.BaristaViewPagerInteractions.swipeViewPagerBack;
+import static com.schibsted.spain.barista.interaction.BaristaViewPagerInteractions.swipeViewPagerForward;
+
+@RunWith(AndroidJUnit4.class)
+public class PartiallyVisibleViewPagerTest {
```

Comment:
I tried running the tests without the workaround (reverting the swipeLeft and swipeRight imports) to see if I can reproduce the error, and they pass too. 
I think I understand the issue and I agree it should fail in the given scenario. Maybe I'm missing something?

Reply:
That's... weird, I did the same verification as you did and I'd swear they failed but let me give it another try

Reply:
You were right, I don't know if I changed something in the layout in the last minute but it was working with both implementations. I just put the view pager inside a frame layout and looks like I now can reproduce the behavior that led me to this PR :+1: